docs/UserGuide.md

```diff
@@ -384,13 +381,13 @@ A student can have any number of tags (including 0).
 
 Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 pp/92345678 pe/jackdoe@example.com sch/John's School stream/John stream lvl/J1`
```

Comment:
```suggestion
* `add n/John Doe p/98765432 e/johnd@gmail.com a/Sengkang Ave 2, Block 123, #02-01 pp/92345678 pe/jackdoe@gmail.com sch/Seng Kang Secondary School stream/O-Level stream lvl/S1` Adds a `S1` student named `John Doe` who is studying `O-Level` at `Seng Kang Secondary School` and lives at `Sengkang Ave 2, Block 123, #02-01`, with phone number `98765432`, email `johnd@gmail.com`, parent contact number `92345678` and parent email `jackdoe@gmail.com`.
```
CS2103 website say example need to be sound like rl example. Not sure if the additional explanation is too lengthy.. But other examples have additional desc